integration/js/README.md

```diff
@@ -33,7 +34,10 @@ chmod +x configureMac.sh
 ./configureMac.sh
 ```
 
-Choose 'y'.
+Choose 'y'. <br />
+Note: If the `./configureMac.sh` script fails then make sure you are using `maven@3.5`. We are not using the latest version of KITE and the setup does not seem to be compatible with the latest version of maven.<br />
```

Comment:
it is maven 3.6.3 as mentioned here https://github.com/aws/amazon-chime-sdk-js/blob/master/integration/js/script/install-kite#L12.
Also this can be shorter as "The KITE version in use does not work with later version of maven".

Reply:
I have updated the documentation to recommend maven version 3.6.3 or earlier. I have marked the maven commands that I added as an example. Maven 3.6 is not available in homebrew.

I have reduced the length of the sentence. Thank you.